modin/pandas/base.py

```diff
@@ -1875,15 +1896,18 @@ def mask(
         """
         Replace values where the condition is True.
         """
-        return self._default_to_pandas(
-            "mask",
-            cond,
-            other=other,
+        return self._create_or_update_from_compiler(
```

Comment:
Why are we using this instead of just the constructor? Or will this method just use the constructor if in place is false, and otherwise do an in place update?

Reply:
Indeed. It handles the `inplace` properly.